samples/README.md

```diff
@@ -1,3 +1,15 @@
+# Sample apps for the AWS IoT Device SDK for Embedded C
+
+All samples are written in C unless otherwise mentioned. The these sample apps are included in the SDK and described below.
+ * [`subscribe_publish_sample`](#subscribe-publish-sample) - a simple pub/sub MQTT example
+ * [`subscribe_publish_cpp_sample`](#subscribe-publish-cpp-sample) - a simple pub/sub MQTT example written in C++
```

Comment:
Is there a CPP sample - I do not see it. Even the Makefile does not have anything related to C++: https://github.com/aws/aws-iot-device-sdk-embedded-C/blob/master/samples/linux/subscribe_publish_sample/Makefile

Reply:
Removed references to non-existent sample.